modules/distribution/src/repository/resources/dbscripts/apimgt/oracle.sql

```diff
@@ -768,7 +767,7 @@ CREATE TABLE FIDO_DEVICE_STORE (
         TENANT_ID INTEGER,
         DOMAIN_NAME VARCHAR(255) NOT NULL,
         USER_NAME VARCHAR(45) NOT NULL,
-    TIME_REGISTERED TIMESTAMP,
+	TIME_REGISTERED TIMESTAMP,
```

Comment:
Directly copied content from https://github.com/wso2/carbon-apimgt/tree/v5.0.4/features/apimgt/org.wso2.carbon.apimgt.core.feature/src/main/resources/sql as apim version upgraded to v5.04